pkg/networkplugin-syncer/handlers/ovn/uninstall.go

```diff
@@ -0,0 +1,162 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ovn
+
+import (
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/libovsdbops"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
+	"k8s.io/klog/v2"
+)
+
+func (ovn *SyncHandler) Stop(uninstall bool) error {
+	if !uninstall {
+		return nil
+	}
+
+	klog.Infof("Uninstalling Submariner changes from the node")
+
+	// Delete the submariner logical router, ports and flows
+	staleLRSRPred := func(item *nbdb.LogicalRouterStaticRoute) bool {
+		return item.OutputPort != nil && *item.OutputPort == submarinerUpstreamRPort
+	}
+
+	err := libovsdbops.DeleteLogicalRouterStaticRoutesWithPredicate(ovn.nbdb, submarinerLogicalRouter, staleLRSRPred)
+	if err != nil {
+		klog.Errorf("failed to delete ovn static routes for port: %s due to : %v", submarinerUpstreamRPort, err)
```

Comment:
```suggestion
		klog.Errorf("Failed to delete ovn static routes for port: %s due to : %v", submarinerUpstreamRPort, err)
```

Although we do not enforce this. AFAICT, the common pattern we follow is to use capital letter when printing the error message and use small letters when returning the error.

Reply:
Done.